### PR TITLE
Yajl

### PR DESCRIPTION
**Pull request by tsmolka**
Added yajl package. Compiles and seems to work fine.
- http://lloyd.github.io/yajl/
- https://www.archlinux.org/packages/extra/i686/yajl/

### CODE REVIEW

File: mingw-w64-yajl/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
+_realname=yajl
+if [ ! -z "${MINGW_PACKAGE_PREFIX}" ]; then
+    pkgprefix="${MINGW_PACKAGE_PREFIX}-"
+    pkgdirprefix="${MINGW_PREFIX}/"
+    cmakegen="MSYS Makefiles"
+else
```
Comment (Alexpux) [inline mingw-w64-yajl/PKGBUILD]: Remove it. We have separate repo for mingw and msys packages.

File: mingw-w64-yajl/01-fix-compiler.patch [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
++IF (CMAKE_C_COMPILER_ID EQUAL "MSVC")
+   SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /W4")
+   ADD_DEFINITIONS(-DWIN32)
+   SET(linkFlags "/PDB:NONE /INCREMENTAL:NO /OPT:NOREF /OPT:NOICF")
+@@ -46,7 +46,7 @@
+   SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /wd4996 /wd4255 /wd4130 /wd4100 /wd4711")
+   SET(CMAKE_C_FLAGS_DEBUG "/D DEBUG /Od /Z7")
+   SET(CMAKE_C_FLAGS_RELEASE "/D NDEBUG /O2")
+-ELSE (WIN32)
++ELSE (CMAKE_C_COMPILER_ID EQUAL "MSVC")
```
Comment (Alexpux) [inline mingw-w64-yajl/01-fix-compiler.patch]: You can write just ````IF(MSVC)````

**Comment by Alexpux** [comment]
Will my comments be fixed?

**Comment by tsmolka** [comment]
Sorry for the delay. Updated.

**Pull request merged**